nltk/parse/stanford.py

```diff
@@ -281,6 +282,15 @@ class StanfordParser(GenericStanfordParser):
 
     _OUTPUT_FORMAT = 'penn'
 
+    def __init__():
+        warnings.simplefilter('always', DeprecationWarning)
```

Comment:
I'm not sure whether `warnings.simplefilter` should be called here. It should be up to the user to decide what warnings are shown.